docs/installation-vagrant.rst

```diff
@@ -0,0 +1,119 @@
+Kitsune via Vagrant
+================
+
+Core developers run Kitsune in a `Vagrant`_-managed virtual machine so we can run
+the entire Kitsune stuck.
+If you're on Mac OS X or Linux and looking for a quick way to get started, you
+should try these instructions.
+
+.. note:: **If you have problems getting vagrant up**, check Errors_ below.
+
+.. _vagrant: http://vagrantup.com/
+.. _uses NFS to share the current working directory: http://docs.vagrantup.com/v2/synced-folders/nfs.html
+
+Install and run everything
+--------------------------
+
+#. Install VirtualBox 4.x from http://www.virtualbox.org/
+
+   .. note:: (Windows) After installing VirtualBox you need to set
+              PATH=C:\\Program Files\\Oracle\\VirtualBox\\VBoxManage.exe;
+
+#. Install vagrant >= 1.6 using the installer from `vagrantup.com <http://vagrantup.com/>`_
+
+#. Clone Kitsune::
+
+       git clone https://www.github.com/mozilla/kitsune.git
+       cd kitsune
+
+#. Start the VM and install everything. (approx. 30 min on a fast net connection).::
+
+      vagrant up
+
+   .. note:: VirtualBox creates VMs in your system drive.
+             If it won't fit on your system drive, you will need to `change that directory to another drive <http://emptysquare.net/blog/moving-virtualbox-and-vagrant-to-an-external-drive/>`_.
+
+   At the end, you should see::
+
+      => default: notice: Finished catalog run in .... seconds
+
+
+   If the above process fails with an error, check `Errors`_.
+
+#. Log into the VM with ssh::
+
+       vagrant ssh
+
+#. Create a Superuser::
+
+       cd kitsune
+       source venv/bin/activate
+       ./manage.py createsuperuser
+   You should give the Username and Password for login into your local Kitsune.
+   Your password must contain letters and numbers
+
+#.  Run Kitsune::
+
+       ./manage.py runserver 0.0.0.0:8000
+
+Now, just navigate to <http://localhost:8000> to see the application!
+
+
+.. _Errors:
+
+Errors during `vagrant up`
+--------------------------
+
+``vagrant up`` starts the virtual machine. The first time you run ``vagrant up`` it
+also `provisions <https://docs.vagrantup.com/v2/cli/provision.html>`_ the vm -
+i.e., it automatically installs and configures Kitsune software on the vm. We
+provision the vm with `vagrant_provision in the `/bin directory
+<https://github.com/mozilla/kuma/tree/master/bin>`_.
+
+Sometimes we put vagrant declarations in the wrong order. Which means some
```

Comment:
What does "Sometimes we put vagrant declarations in the wrong order." mean? Can you clarify this? Wouldn't it be better to eliminate any problems with the Vagrant declarations, instead of this work-around?